bazel/format/clang_tidy/run_clang_tidy.sh

```diff
@@ -21,4 +21,6 @@ truncate -s 0 "$OUTPUT"
 # place it in the current directory
 test -e .clang-tidy || ln -s -f "$CONFIG" .clang-tidy
 
+# echo "RUN: ${CLANG_TIDY_BIN} ${*}"
+
```

Comment:
leftover?

Reply:
i figured keep it - its kinda useful for debugging